utilities/decrypt_data/decryption_for_program_service.py

```
@@ -0,0 +1,127 @@
+import time
+import uuid
+import logging
+import requests
+import psycopg2
+from datetime import datetime
+from psycopg2.extras import Json, DictCursor
+
+# ——— Configuration ——————————————————————————————————————————————
+CONFIG = {
+    "db": {
+        "dsn": ""
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)